src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/command.html.twig

```diff
@@ -0,0 +1,251 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block menu %}
+    <span class="label">
+        {# todo set icon #}
+{#        <span class="icon">{{ source('@WebProfiler/Icon/command.svg') }}</span>#}
```

Comment:
terminal-2 in tabler icons :)

Reply:
I'm sorry I don't understand this comment :).

Reply:
Ho sorry, my bad
From what ive understood, the new design of profiler (on which this PR was based also) has began to introduce some view/icon coming from the tabler project and its icon https://tabler-icons.io

Reply:
I've missed that, thanks! Will do.

Reply:
Done :). 🎉